roles/install_openstack_ca/molecule/default/prepare.yml

```diff
@@ -0,0 +1,62 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Prepare
+  hosts: all
+  vars:
+    cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
+    cifmw_openshift_kubeconfig: "{{ ansible_user_dir  }}/.crc/machines/crc/kubeconfig"
+  pre_tasks:
+    - name: Ensure CRC is started
+      async: 1800
+      poll: 0
+      register: _crc_start
+      ansible.builtin.command:
+        cmd: crc start
+  roles:
+    - role: test_deps
+    - role: ci_setup
+    - role: install_yamls
+    - role: ci_local_storage
+  tasks:
+    - name: Check for CRC status
+      ansible.builtin.async_status:
+        jid: "{{ _crc_start.ansible_job_id  }}"
+      register: _crc_status
+      until: _crc_status.finished
+      retries: 100
+      delay: 10
+
+    - name: Install install_yamls dependencies
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: "make_download_tools"
+
+    - name: Install openstack operator
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: "make_openstack_wait"
+      vars:
+        make_openstack_wait_params:
+          BMO_SETUP: false
+          NETWORK_ISOLATION: false
+
+    - name: Deploy openstack controlplane and wait for its completion
+      vars:
+        sample_path: "{{ ansible_user_dir  }}/src/github.com/openstack-k8s-operators/ci-framework/roles/install_openstack_ca/molecule/default/files/minimal_openstack_controlplane.yml"
```

Comment:
Seems this file is missing from the patch causing CI to fail: https://logserver.rdoproject.org/36/1136/df169fef3a60864d7bd93f6a68f3a2f737b615af/github-check/cifmw-molecule-install_openstack_ca/126a301/job-output.txt

```
2024-02-13 15:28:06.438932 | controller | TASK [Deploy openstack controlplane and wait for its completion] ***************
2024-02-13 15:28:16.294960 | controller | fatal: [instance]: FAILED! => changed=true
2024-02-13 15:28:16.295294 | controller |   cmd:
2024-02-13 15:28:16.295569 | controller |   - oc
2024-02-13 15:28:16.295847 | controller |   - apply
2024-02-13 15:28:16.296173 | controller |   - -f
2024-02-13 15:28:16.296473 | controller |   - /home/zuul/src/github.com/openstack-k8s-operators/ci-framework/roles/install_openstack_ca/molecule/default/files/minimal_openstack_controlplane.yml
2024-02-13 15:28:16.296739 | controller |   delta: '0:00:09.579727'
2024-02-13 15:28:16.297073 | controller |   end: '2024-02-13 15:28:16.247570'
2024-02-13 15:28:16.297346 | controller |   msg: non-zero return code
2024-02-13 15:28:16.297649 | controller |   rc: 1
2024-02-13 15:28:16.298069 | controller |   start: '2024-02-13 15:28:06.667843'
2024-02-13 15:28:16.298296 | controller |   stderr: 'error: the path "/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/roles/install_openstack_ca/molecule/default/files/minimal_openstack_controlplane.yml" does not exist'
2024-02-13 15:28:16.298565 | controller |   stderr_lines: <omitted>
2024-02-13 15:28:16.298831 | controller |   stdout: ''
2024-02-13 15:28:16.299255 | controller |   stdout_lines: <omitted>
2024-02-13 15:28:16.299709 | controller |
2024-02-13 15:28:16.300093 | controller | PLAY RECAP *********************************************************************
2024-02-13 15:28:16.300440 | controller | instance                   : ok=49   changed=26   unreachable=0    failed=1    skipped=25   rescued=0    ignored=0
2024-02-13 15:28:16.300782 | controller |
2024-02-13 15:28:16.392364 | controller | CRITICAL Ansible return code was 2, command was: ['ansible-playbook', '--inventory', '/home/zuul/.cache/molecule/install_openstack_ca/default/inventory', '--skip-tags', 'molecule-notest,notest', '/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/roles/install_openstack_ca/molecule/default/prepare.yml']
```

Reply:
yep, I pushed the changes quickly last night and I forgot to add it... I'll update the patch now